src/components/VideoLayout/VideoLayout.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import styles from './VideoLayout.module.scss';
+
+type Props = {
+  inlineLayout: boolean;
+  videoDetails: React.ReactNode;
+  videoDetailsInline: React.ReactNode;
+  inlinePlayer: React.ReactNode;
+  cinemaPlayer: React.ReactNode;
+  relatedVideosGrid: React.ReactNode;
+  relatedVideosList: React.ReactNode;
+};
+
+const VideoLayout: React.FC<Props> = ({
+  inlineLayout,
+  videoDetails,
+  videoDetailsInline,
+  inlinePlayer,
+  cinemaPlayer,
+  relatedVideosList,
+  relatedVideosGrid,
+}: Props) => {
+  if (inlineLayout) {
+    return (
+      <div className={styles.videoInlineLayout}>
+        <div className={styles.player}>{inlinePlayer}</div>
+        <div className={styles.relatedVideosList}>{relatedVideosList}</div>
+        <div className={styles.videoDetailsInline}>{videoDetailsInline}</div>
+      </div>
+    );
+  }
+
+  return (
+    <div className={styles.videoCinemaLayout}>
+      {cinemaPlayer}
+      {videoDetails}
```

Comment:
I think with minimal effort, playlistlivechannel can also use VideoLayout (always with inline=false), then you can fold VideoDetails and VideoDetailsInline into this as well.  I appreciate component composition, but in this case I'm worried it's causing more duplication than necessary, because most things only need to be used in 1 place.

Reply:
I thought of adding the VideoDetail and VideoDetailsInline to the video layout, but it would make the video layout less flexible. But I will change it.